client/src/index.js

```diff
@@ -1,11 +1,10 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
-import App from "../src/modules/core/App";
+import App from "./modules/core/App.component";
 import { BrowserRouter } from "react-router-dom";
```

Comment:
This is a node module import! this imports will go to the top of the import list